experimental/saitypesextensions.h

```diff
@@ -74,6 +74,8 @@ typedef enum _sai_object_type_extensions_t
 
     SAI_OBJECT_TYPE_HA_SCOPE,
 
+    SAI_OBJECT_TYPE_ROUTING_GROUP,
```

Comment:
why this is not added at the end ? such changes will not be backward compatible later

Reply:
this is the same problem as the API type. we need to reset the generated SAI header before generating the new one.

Reply:
Moved to the end thanks to new api gen infra.